android/PhysicalWeb/app/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,10 @@
     <string name="fatbeacon_notification_title">Physical Web is Sharing as a FatBeacon</string>
     <string name="wifi_direct_notification_title">Physical Web is sharing with WifiDirect</string>
     <string name="demos">Demos</string>
-    <string name="fat_beacon_demo_title">Fat Beacon Demo</string>
+    <string name="fat_beacon_demo_title">FatBeacon Demo</string>
     <string name="fat_beacon_demo_summary">Runs a hello world example of FatBeacon. It will broadcast over ble and show up in another Physical Web app that has FatBeacon enabled. Its title will be Hello World.</string>
+    <string name="wifi_direct_demo_title">Wi-Fi Direct Demo</string>
+    <string name="wifi_direct_demo_summary">Runs a hello world example of Wi-Fi Direct. It will broadcast over Wi-Fi Direct and show up in another Physical Web app that has Wi-Fi Direct enabled. Its title will be Hello World. For best reliability use two devices running Android N.</string>
```

Comment:
These demo summaries could be more informative.  For instance, it's not clear that this demo turns your device into a beacon, which is why you need two devices.  We should also mention the offline webpage (that's the coolest part!)

Maybe something like (feel free to modify):

Wi-Fi Direct Demo

Turns this device into a Wi-Fi Direct beacon advertising an offline webpage.  To see the beacon, install the Physical Web app on a second device and enable Wi-Fi Direct in the app settings.  For best reliability, use two devices running Android Nougat.